packages/data-explorer/src/repository/dataRepository.ts

```diff
@@ -140,8 +140,13 @@ const getRowDataWithReferenceLabels = async (tableId: string, rowId: string, met
   return levelOneRowMapper(response.data, metaData)
 }
 
+const deleteRow = async (tableId: string, rowId: string) => {
+  return axios.delete(`/api/data/${tableId}/${rowId}`)
```

Comment:
Should there be state handeling if delete fails?... maybe the user is not allowed to delete this row and should be notified?

Reply:
yes , should be passed to the action and handled there imo